get-forms/index.html

```diff
@@ -3,11 +3,31 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>GitHub Search Form</title>
     <meta name="description" content="My Github Search form" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
-    <form action="https://github.com/search" method="get"></form>
+    <form action="https://github.com/search" method="get">
+      <div>
+        <label for="search">Search GitHub</label>
+        <input type="text" name="q" id="search"/>
+        <button type="submit">Search</button>
+      </div>
+      <div>
+        <label for="sort">Sort by</label>
+        <select name="s" id="sort">
+          <option value="stars">Stars</option>
+          <option value="fork">Fork</option>
+          <option value="updated">Updated</option>
+        </select>
+      </div>
+      <div>
+        <label for="order1">Ascending</label>
+        <input type="radio" name="o" value="asc" id="order1"/>
+        <label for="order2">Descending</label>
+        <input type="radio" name="o" value="dsc" id="order2"/>
+      </div>
+    </form>
```

Comment:
Well done, this is an excellent first attempt

Reply:
Thank you Fathi, Yeah the SEO score is low but that is something beyond the curent exercise so as the volunteers said, nothing to worry about right now.